src/xg.hpp

```diff
@@ -224,8 +231,11 @@ class XG {
         }
     };
     
+    int64_t node_height(ThreadMapping node) const;
```

Comment:
This is for reading the (private) h_iv vector since I need this information for my rectangular decompositions and it's much faster to do this than do a search for the interval containing all paths at a node